### DISCUSSION TITLE
Link to pages or collections

### DISCUSSION THREAD

**Discussion by axelrindle** [outline/outline#7690] [Ideas]
I'd like to insert links to other pages or collections without pasting the full url in case a page moves etc.

Cool would be the following:

1. Type **/link** or something
2. Choose a collection or page (searchable)

**Comment by tommoor** [comment]
cmd+k is what you're looking for, we will have this via the @ mention menu soon though

**Reply by axelrindle** [reply]
Ah great, I should use this shortcut more often :smile: While that allows me to link to a page, I can't link an entire collection. Is that planned? I'd try to send a PR otherwise.

**Reply by al-tr** [reply]
`@` mention for the other pages linking is very intuitive (notion and others do this), and handy on mobile since there's really no "cmd+k" there.
@tommoor could you please link the issue/pr/discussion to track this feature release?

**Reply by tommoor** [reply]
Tracked as #7680
